FIRBoundaries.dat

```diff
@@ -24113,7 +24119,7 @@ UKBV|0|0|93|47.766667|27.433333|52.364333|34.450000|50.065500|30.941667
 51.240761|34.361997
 51.244200|34.384344
 51.242419|34.428728
-UKFV|0|0|33|42.733333|30.150000|47.733333|37.368333|45.233333|33.759167
+URFV|0|0|33|42.733333|30.150000|47.733333|37.368333|45.233333|33.759167

```

Comment:
Intended as per #77 and #64. Please undo this change.

Reply:
The reason i did this change is because next days VATRUS has an event focused on URFF and i was thinking that we need to change it to let pilots know that this area is online that day. If we leave it UKFF noone will know and unfortunately from the moment this icao does not exist , no metar is shown. what do you think guys?

Reply:
As stated in their websites and the issues mentioned above, this should indeed be UKFV.

P.D.: How does an FIR have a METAR?

Reply:
im talking about the airport of Simferopol UKFF/URFF not the FIR. from the moment this area in real life is under the russian ACC then all the icao codes have been renamed to URXX. So the metar will show under URFF only. So my proposal is to leave UKFV CTR as it is and at least rename the airport to URFF? how does it sound?

Reply:
Only in the russian AIP have the names been ammended to URFF, all other authorities like ICAO, IATA, Navigraph (but also vatsim, vatrus and vatukraine) still use UKFF.

Reply:
Ok. should i change it back to default?

Reply:
Please do! 😛